spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -240,12 +249,23 @@ public SparkContext createSparkContext() {
       }
     }
 
-    SparkConf conf =
-        new SparkConf()
+    SparkConf conf = new SparkConf()
             .setMaster(getProperty("master"))
             .setAppName(getProperty("spark.app.name"))
             .set("spark.repl.class.uri", classServerUri);
 
+    if (useCassandraContext()) {
+      conf.set("spark.cassandra.connection.host", getProperty("spark.cassandra.connection.host"));
+
+      if (getProperty("spark.cassandra.auth.username") != null) {
+        conf.set("spark.cassandra.auth.username", getProperty("spark.cassandra.auth.username"));
+      }
+
+      if (getProperty("spark.cassandra.auth.password") != null) {
+        conf.set("spark.cassandra.auth.password", getProperty("spark.cassandra.auth.password"));
+      }
+    }
```

Comment:
I think line 268 sets all property into conf that property starts with 'spark.'. So isn't it duplicated?